spi/common/boot-spi/src/main/java/org/eclipse/edc/spi/monitor/ConsoleMonitor.java

```diff
@@ -82,6 +88,27 @@ public void debug(Supplier<String> supplier, Throwable... errors) {
         output(DEBUG, supplier, errors);
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I don't think we should add equal/hashCode implementation for "behavior" classes.
The reason here looks to be testing, a better reason to not do that.
With assertJ is actually possible to check equality on private fields via `extracting` method

Reply:
[73f5085](https://github.com/eclipse-edc/Connector/pull/4476/commits/73f5085ddf516f1d8b29ec1c45e9c2c737ac0c90)